.github/workflows/release.yaml

```diff
@@ -0,0 +1,144 @@
+            core.setOutput("release", release);
+            core.setOutput("version", version);
+            core.setOutput("tag", latestRelease.data.tag_name);
+
+      - name: Get version and release Tag
+        id: get_tags
+        run: |
+          if [ "${{ github.event_name }}" == "workflow_dispatch" ]; then
+            RELEASE=${{ steps.get_latest_release.outputs.release }}
+            TAG=${{ steps.get_latest_release.outputs.tag }}
+            if [ -z ${{ steps.get_latest_release.outputs.version }} ]; then
+              VERSION=${TAG#v}
+            else
+              VERSION=${{ steps.get_latest_release.outputs.version }}
+            fi
+          else
+            RELEASE=1
+            TAG=${{ github.event.release.tag_name }}
+            VERSION=${TAG#v}
+          fi
+          echo "VERSION=$VERSION" >> $GITHUB_OUTPUT
+          echo "RELEASE=$RELEASE" >> $GITHUB_OUTPUT
+          echo "TAG=$TAG" >> $GITHUB_OUTPUT
+
+  build-rpm:
+    needs: gather-informations
+    uses: alumet-dev/packaging/.github/workflows/build_rpm.yaml@main
+    with:
+      target-architecture: x86_64
+      build-version: ${{ needs.gather-informations.outputs.version }}
+      release-version: ${{ needs.gather-informations.outputs.release }}
+      tag: ${{ needs.gather-informations.outputs.tag }}
+
+  attach-artifacts:
+    needs:
+      - build-rpm
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Download RPM artifact
+        uses: actions/download-artifact@v4
+        with:
+          pattern: '**/*.rpm'
+          path: ./rpm
+
+      - name: Create new release
+        if: ${{ github.event_name != 'workflow_dispatch' }}
+        uses: softprops/action-gh-release@v2
+        with:
+          files: |
+            rpm/**/*.rpm
+
+      - name: Update release, deleting old artifacts
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        id: get_latest_release_and_delete_old_artifacts
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const latestRelease = await github.rest.repos.getLatestRelease({
+              owner: context.repo.owner,
+              repo: context.repo.repo
+            });
+
+            let id = latestRelease.data.id;
+            let tag = latestRelease.data.tag_name;
+
+            const assets = await github.rest.repos.listReleaseAssets({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              release_id: id
+            });
+
+            for (const asset of assets.data) {
+              try {
+                if ( asset.name.endsWith(".deb") || asset.name.endsWith(".rpm")) {
+                  await github.rest.repos.deleteReleaseAsset({
+                    owner: context.repo.owner,
+                    repo: context.repo.repo,
+                    asset_id: asset.id
+                  });
+                } catch (error) {
```

Comment:
it fails see error here https://github.com/ahuret/alumet/actions/runs/14039822895/job/39307258080#step:5:51